libs/community/langchain_community/vectorstores/azure_cosmos_db_no_sql.py

```diff
@@ -34,9 +62,14 @@ def __init__(
         indexing_policy: Dict[str, Any],
         cosmos_container_properties: Dict[str, Any],
         cosmos_database_properties: Dict[str, Any],
+        full_text_policy: Optional[Dict[str, Any]] = None,
         database_name: str = "vectorSearchDB",
         container_name: str = "vectorSearchContainer",
+        text_key: str = "text",
+        embedding_key: str = "embedding",
+        metadata_key: str = "metadata",
         create_container: bool = True,
+        full_text_search_enabled: bool = False,
```

Comment:
```suggestion
        database_name: str = "vectorSearchDB",
        container_name: str = "vectorSearchContainer",
        create_container: bool = True,
        *,
        full_text_policy: Optional[Dict[str, Any]] = None,
        text_key: str = "text",
        embedding_key: str = "embedding",
        metadata_key: str = "metadata",
        full_text_search_enabled: bool = False,
```
ordering change also breaking

Reply:
nvm these are kwargs